mouse_activity_client.py

```diff
@@ -0,0 +1,90 @@
+"""
+TCP-client collect the information about mouse activity on PC
+every 60 seconds and send in on the server.
+
+For cancel the program running press Ctrl+X.
+"""
+import keyboard
+import socket
+import time
+from pynput import mouse
+
+
+sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
```

Comment:
In this case, we open a connection with the TCP-server for all time of working your script and working `mouse_activity_info`. I suppose that an effective approach makes connect only when we need to send data (once time in interval). And you can try to use `context manager` for the socket.

Reply:
Fixed (and in `traffic_info_client.py` too).
P.S. If a want to make the same when i use the classes for collectors, i recieve too many notifications `Instance attribute <some attribute> defined outside __init__`. How could i resolve this? If i need to put all the attributes in the `__init__` method, what values should i initialize them to (especially a socket object)?

Reply:
It depends on the aim of this attribute, for socket objects we can use `None` for example